proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress.md

```diff
@@ -45,119 +44,126 @@ If the SDL app or HU cannot use this function, it will perform the same operatio
-    <description>Sent by SDL in UpdateDeviceList. 'true' - if device is allowed for PolicyTable Exchange; 'false' - if device is NOT allowed for PolicyTable Exchange </description>
-  </param>
-+ <param name="bluetoothDeviceAddress" type="String" mandatory="false">
-+   <description>Device BT Address</description>
-+ </param>
-</struct>
-...
-<struct name="HMIApplication">
-...
-    <param name="deviceInfo" type="Common.DeviceInfo" mandatory="true">
-       <description>The ID, serial number, and transport type by which the named app's device is connected to HU.</description>
-    </param>
-...
-    <param name="cloudConnectionStatus" type="Common.CloudConnectionStatus" mandatory="false"></param>
-+    <param name="requiresAudioSupport" type="Boolean" mandatory="false">
-+        <description>Set whether or not this app requires the use of an audio streaming output device.</description>
-+    </param>
-</struct>
+  <struct name="DeviceInfo">
+      <param name="name" type="String" mandatory="true">
+          <description>The name of the device connected.</description>
+      </param>
+      <param name="id" type="String" mandatory="true">
+          <description>The ID of the device connectedi: either hash of device's USB serial number(in case of USB connection) or has of device's MAC address(in case of BlueTooth or WIFI connection</description>
+      </param>
+      <param name="transportType" type="Common.TransportType" mandatory="false">
+         <description>The transport type the named-app's-device is connected over HU(BlueTooth, USB or WiFi). It must be provided in OnAppRegistered and in UpdateDeviceList</description>
+      </param>
+      <param name="isSDLAllowed" type="Boolean" mandatory="false">
+          <description>Sent by SDL in UpdateDeviceList. 'true' - if device is allowed for PolicyTable Exchange; 'false' - if device is NOT allowed for PolicyTable Exchange </description>
+      </param>
++     <param name="bluetoothDeviceAddress" type="String" mandatory="false">
++         <description>Device BT Address</description>
++     </param>
+  </struct>
+  ...
+  <struct name="HMIApplication">
+  ...
+      <param name="deviceInfo" type="Common.DeviceInfo" mandatory="true">
+          <description>The ID, serial number, and transport type by which the named app's device is connected to HU.</description>
+      </param>
+  ...
+      <param name="cloudConnectionStatus" type="Common.CloudConnectionStatus" mandatory="false"></param>
++     <param name="requiresAudioSupport" type="Boolean" mandatory="false">
++         <description>Set whether or not this app requires the use of an audio streaming output device.</description>
++     </param>
+  </struct>
 ```
 
+### MediaStreamingStatus class
+
+The library supports the new version of the protocol specification. However the app needs to use `MediaStreamingStatus` class before registering if the IVI system uses the earlier version.
```

Comment:
Should this be re-worded, per [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/941#issuecomment-644850433):
```suggestion
If the Java Suite library supports the newer version of the protocol specification, but the IVI system is using a lower version, the app will still need to use the `MediaStreamingStatus` class before attempting to register.
```